### PR TITLE
Xcode 10.2 Support

### PR DESCRIPTION
**Pull request by stephencelis**
This gets all dependencies, etc., in order so we can finally `rm` Xcode 10.1.

### CODE REVIEW

File: PointFree.playground/Pages/Database.xcplaygroundpage/Contents.swift [hunk 1]
```diff
@@ -7,28 +7,28 @@ PlaygroundPage.current.needsIndefiniteExecution = true
...
-  .perform()
-  .right!
-
-let updatedUser = Current.database.fetchUserById(user.id)
-  .run
-  .perform()
-  .right!!
-
-updatedUser.episodeCreditCount
+//Current.database.migrate().run.perform()
```
Comment (mbrandonw) [inline PointFree.playground/Pages/Database.xcplaygroundpage/Contents.swift]: should we revert this?

File: Sources/PointFree/Account/Account.swift [hunk 2]
```diff
@@ -40,7 +40,7 @@ private func fetchAccountData<I>(
 
   let (user, subscriberState) = lower(conn.data)
 
-  let userSubscription = user.subscriptionId
+  let userSubscription: EitherIO<Error, Models.Subscription> = user.subscriptionId
```
Comment (mbrandonw) [inline Sources/PointFree/Account/Account.swift]: type inference probs?

File: PointFree.playground/Pages/Database.xcplaygroundpage/Contents.swift (hunk 1 above)
Comment (stephencelis) [reply PointFree.playground/Pages/Database.xcplaygroundpage/Contents.swift]: ah yeah. snuck in sorry!

File: Sources/PointFree/Account/Account.swift (hunk 2 above)
Comment (stephencelis) [reply Sources/PointFree/Account/Account.swift]: yeah same as with Swift 5 :( thinks it's an optional

**Pull request merged**